backend/inspirehep/editor/views.py

```diff
@@ -189,12 +189,11 @@ def _simplify_ticket_response(ticket):
 @login_required_with_roles([Roles.cataloger.value])
 def refextract_text():
     """Run refextract on a piece of text."""
-    with local_refextract_kbs_path() as kbs_path:
-        extracted_references = extract_references_from_string(
-            request.json["text"],
-            override_kbs_files=kbs_path,
-            reference_format="{title},{volume},{page}",
-        )
+    extracted_references = extract_references_from_string(
+        request.json["text"],
+        override_kbs_files=create_journal_dict(),
```

Comment:
wrong use of the API here, it should be
```suggestion
        override_kbs_files={"journals": create_journal_dict()},
```
It would be good to have a real integration test to make sure this works, where you
1. create some non-existing journal, e.g.`Journal of Testing` with short title `J.Testing`
2. call this endpoint with a reference such as `John Smith, Journal of Testing 42 (2020) 1234`
3. assert that the output contains `J.Testing`.